examples/terraform/gce/variables.tf

```diff
@@ -90,7 +90,7 @@ variable "control_plane_volume_size" {
 }
 
 variable "control_plane_image_family" {
-  default     = "ubuntu-1804-lts"
+  default     = "ubuntu-2004-lts"
```

Comment:
This is a breaking change because it might cause all instances to be recreated for those who don't bind the image in `terraform.tfvars`. If this is required, I suggest putting a warning in the release note.

Reply:
Okay i put this into another PR than.

Reply:
It's okay to leave the change in this PR, just make sure the release note reflects the change.